apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ir/IrBuilder.kt

```diff
@@ -183,6 +184,7 @@ internal class IrBuilder(
         implements = implementsInterfaces,
         keyFields = schema.keyFields(name),
         description = description,
+        // XXX: this is not spec-compliant. Directive cannot be on object definitions
```

Comment:
I kept those because it shouldn't harm too much but we should not hesitate to remove them if anything comes up.

Reply:
Oh I think that answers my previous comment 😅